config/locales/models/transaction.en.yml

```diff
@@ -30,7 +30,7 @@ en:
         value: Transaction amount
     legend:
       transaction:
-        date: Date transaction was made
+        date: Date of transaction
         providing_organisation: Providing organisation
```

Comment:
I think the translation strings for the removed fields are not being used anywhere else, so we could remove them. (But if we're not sure, it's also ok to leave them in for now.)

Reply:
Yeah, I had a similar thought but I decided to leave them just in case